orchestration/values.schema.json

```diff
@@ -69,10 +69,9 @@
         "stagingData": {
           "type": "object",
           "properties": {
-            "project": { "type": "string" },
-            "dataset": { "type": "string" }
+            "project": { "type": "string" }
           },
-          "required": ["project", "dataset"]
+          "required": ["project"]
```

Comment:
Not much point to the multi-level schema here, we could collapse it to `"stagingProject": { "type": "string" }`

Reply:
Alt: we could add more parameters, see above